portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,23 +39,36 @@
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      comments = new ArrayList<>();
-      comments.add("Comment number 1 from server");
-      comments.add("Comment number 2 from server");
-      comments.add("Comment number 3 from server");
+      Query query = new Query("Comments").addSort("first-name", SortDirection.DESCENDING);
```

Comment:
move magic strings to constants

Reply:
do you mean "first-name" or all of them?